src/Microsoft.DotNet.Wpf/tests/UnitTests/PresentationCore.Tests/PresentationCore.Tests.csproj

```diff
@@ -24,7 +24,8 @@
     <PackageReference Include="xunit.stafact" Version="$(XUnitStaFactPackageVersion)" />
     <PackageReference Include="FluentAssertions" Version="$(FluentAssertionsVersion)" />
     <PackageReference Include="System.Collections.Concurrent" Version="4.3.0" />
-    <PackageReference Include="System.Runtime.Serialization.Formatters" Version="9.0.0-preview.7.24359.11 " />
+    <PackageReference Include="System.Formats.Nrbf" Version="$(SystemFormatsNrbfVersion)" />
+    <PackageReference Include="System.Runtime.Serialization.Formatters" Version="$(SystemRuntimeSerializationFormattersPackageVersion)" />
```

Comment:
Without this line I was getting errors about conflicts of different versions of `System.Runtime.Serialization.Formatters` being used.

Moreover, this test project does not seem to be included in the solution file.

Is it somehow included by the CI?

Reply:
> Moreover, this test project does not seem to be included in the solution file.
> 
> Is it somehow included by the CI?

No, it seems like they were never enabled in main (It seems like they were voluntarily disabled in the PR that added them). I took a shot at enabling them here: dotnet/wpf#9471